database/factories/ModelFactory.php

```diff
@@ -76,6 +78,13 @@
     ];
 });
 
+$factory->define(Setting::class, function ($faker) {
```

Comment:
Seem so your tests rely on it, it makes sens to have it. I suppose! I'm only unsure about the default values - I always expect factory built models to be randomly hydrated (not statically).